pymc/tests/distributions/test_continuous.py

```diff
@@ -1802,6 +1802,46 @@ class TestTruncatedNormalUpperArray(BaseTestDistributionRandom):
     ]
 
 
+class TestZeroSumNormal(BaseTestDistributionRandom):
```

Comment:
That baseclass doesn't make sense for the tests

```suggestion
class TestZeroSumNormal:
```